i2c/24lc32_i2c/example.c

```diff
@@ -0,0 +1,59 @@
+#include <stdio.h>
+#include "pico/stdlib.h"
+#include "hardware/i2c.h"
+#include "24lc32_i2c_lib.h"
+
+// Default I2C address of 24LC32
+#define I2C_ADDRESS 0x50
+
+int main() {
+    stdio_init_all();
+
+    eeprom_init(PICO_DEFAULT_I2C_SDA_PIN, PICO_DEFAULT_I2C_SCL_PIN, 400*1000, i2c_default);
```

Comment:
If you're feeling up to it, it might be nice to also add lines like these https://github.com/raspberrypi/pico-examples/blob/master/i2c/bmp280_i2c/bmp280_i2c.c#L214-L221 so that a) you get a suitable compile-time warning if you try to build it for a [board](https://github.com/raspberrypi/pico-sdk/tree/master/src/boards/include/boards) that doesn't have  `PICO_DEFAULT_I2C_SDA_PIN` and `PICO_DEFAULT_I2C_SCL_PIN` defined and b) running `picotool info` against the built UF2 will tell you what pins it expects to use.

For a similar reason (i.e. `picotool info`), it might also be nice to add https://github.com/raspberrypi/pico-examples/blob/master/i2c/bmp280_i2c/CMakeLists.txt#L11-L12 to your `CMakeLists.txt` :slightly_smiling_face: